src/common/components/cards/rules-with-instances.tsx

```diff
@@ -15,6 +14,8 @@ import { CollapsibleComponentCardsProps } from './collapsible-component-cards';
 import { RuleContent, RuleContentDeps } from './rule-content';
 import * as styles from './rules-with-instances.scss';
 
+export const ruleGroupAutomationId = 'cards-rule-group';
```

Comment:
Any reason we can't move this to the place where the automation id is used (as in the collapsible cards component)?